marklogic-data-hub-spark-connector/src/test/java/com/marklogic/hub/spark/sql/sources/v2/WriteTemporalDataTest.java

```diff
@@ -0,0 +1,125 @@
+package com.marklogic.hub.spark.sql.sources.v2;
+
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.marklogic.client.io.DocumentMetadataHandle;
+import com.marklogic.mgmt.resource.databases.DatabaseManager;
+import com.marklogic.mgmt.util.ObjectMapperFactory;
+import org.apache.spark.sql.catalyst.InternalRow;
+import org.apache.spark.sql.sources.v2.writer.DataWriter;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class WriteTemporalDataTest extends AbstractSparkConnectorTest {
+
+    @BeforeEach
+    void setUp() {
+        runAsDataHubDeveloper();
+        addFieldAndIndexes();
```

Comment:
The setup and tear down seems to duplicate code from TemporalWriteTest. Can they be reused ?